<?xml version="1.0"?>
<launch>
  <!-- Cloud Bridge Node - SIMPLIFIED WORKING VERSION -->
  
  <node name="cloud_bridge_node" 
        pkg="elderly_bot" 
        type="cloud_bridge_node.py"
        output="screen"
        respawn="true"
        respawn_delay="10">
    <rosparam command="load" file="$(find elderly_bot)/config/cloud_config.yaml" />
  </node>
  
</launch>